packages/core/src/components/sections/ProductShelf/ProductShelf.tsx

```diff
@@ -6,7 +6,8 @@ import ProductShelfSkeleton from 'src/components/skeletons/ProductShelfSkeleton'
 import { useProductsQuery } from 'src/sdk/product/useProductsQuery'
 import type { ProductsQueryQueryVariables } from '@generated/graphql'
 
-import ProductCard from '../../product/ProductCard'
+import { Components } from './Overrides'
+const { ProductCard } = Components
```

Comment:
What about export default the Overrides to avoid 2 lines to deal with destructuring?